pisp_app/lib/Screens/FindPayee.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import 'package:progress_hud/progress_hud.dart';
+import 'package:pispapp/MockData/Account.dart';
+import 'package:pispapp/log_printer.dart';
+import 'package:pispapp/Screens/TransactionAmount.dart';
+
+class FindPayee extends StatefulWidget {
```

Comment:
Can we call this Lookup instead of Find? Lookup is a more universal term and Find is a little ambiguous in most contexts.

Reply:
Done